test/unit_Polytope.jl

```diff
@@ -112,12 +112,21 @@ for N in [Float64, Rational{Int}, Float32]
     @test BallInf(N[0, 0], N(1)) ⊆ P
     @test !(BallInf(N[0, 0], N(1.01)) ⊆ P)
 
-    # concrete linear map
-    linear_map(N[2 3; 1 2], P)  # invertible matrix
+    # =====================
+    # Concrete linear map
+    # =====================
+    linear_map(N[2 3; 1 2], P) # invertible matrix
     if test_suite_polyhedra
         linear_map(N[2 3; 0 0], P)  # noninvertible matrix
     end
 
+    M = N[2 1; 0 1]
+    linear_map(M, P)
```

Comment:
Use `H` instead of `P` for the tests to also work with `N = Rational{Int}`.

Reply:
It didn't help, because the new keyword argument is defined for polytopes, not for `BallInf` 😢 . So i'll just put this inside a `N <: AbstractFloat` check..

Reply:
The problem is that `an_element` is not type-stable: https://github.com/JuliaReach/LazySets.jl/issues/1105 The `linear_map`s for Rational are built properly.